xayn-ai/src/coi/stats.rs

```diff
@@ -167,9 +175,9 @@ mod tests {
         assert!(relevances[cois[0].id].is_coi());
         assert!(relevances[cois[1].id].is_coi());
         assert!(relevances[cois[2].id].is_coi());
-        assert_approx_eq!(f32, relevances[cois[0].id], 0.5, epsilon = 1e-5);
-        assert_approx_eq!(f32, relevances[cois[1].id], 0.6666667, epsilon = 1e-5);
-        assert_approx_eq!(f32, relevances[cois[2].id], 0.8333333, epsilon = 1e-5);
+        assert_approx_eq!(f32, relevances[cois[0].id], 0.16666646, epsilon = 1e-5);
```

Comment:
while you are at it, since all computations use the same `now` time now, can you please try to reduce the epsilon in all tests of the module to `1e-6` and check if they still pass? if not we can keep epsilon at `1e-5`, but the result might be more precise now, because we don't use different `now`s anymore.